services/src/test/java/se/inera/intyg/common/services/texts/IntygTextsServiceImplTest.java

```diff
@@ -94,4 +95,11 @@ public void shallReturnTrueIfLatestMajorVersionButDifferentMinorVersion() {
         final var actual = service.isLatestMajorVersion("LISJP", "2.0");
         assertTrue(actual);
     }
+
+    @Test
+    public void shallReturnTrueWhenLatestVersionIsNullAsForFk7263() {
```

Comment:
Could consider the naming of the test to something more general....  If no latest version is found it consider the version checked as latest. This will be true for fk7263 but also for any old domain certificate and a situation where a latest version is not found for any reason... 

Decide if you want to consider this feedback... I will approve the PR anyway.

Reply:
The test name has been updated